packages/flame/lib/src/text.dart

```diff
@@ -1,41 +1,46 @@
-import 'dart:ui';
+import 'dart:ui' hide TextStyle, TextDirection;
 
-import 'package:flutter/material.dart' as material;
+import 'package:flutter/material.dart';
 
 import 'anchor.dart';
 import 'components/cache/memory_cache.dart';
 import 'components/text_component.dart';
 import 'extensions/size.dart';
 import 'extensions/vector2.dart';
 
-/// [TextRenderer] is the abstract API that Flame uses for rendering text in its features
-/// this class can be extended to provide an implementation of text rendering in the engine.
+/// [TextRenderer] is the abstract API that Flame uses for rendering text.
+/// This class can be extended to provide another implementation of text
+/// rendering in the engine.
 ///
 /// See [TextPaint] for the default implementation offered by Flame
-abstract class TextRenderer<T extends BaseTextConfig> {
+abstract class TextRenderer<T extends TextStyle> {
```

Comment:
Since the text renderer is not responsible for its config file, maybe this generic is not needed too?

Reply:
True, that argument should be removed completely.